pandas/tests/test_downstream.py

```diff
@@ -53,6 +55,15 @@ def test_xarray(df):
     assert df.to_xarray() is not None
 
 
+def test_xarray_oo_optimizable():
```

Comment:
you can remove the "xarray" here, as it is generally useful to ensure this, not only for xarray